includes/defaults.inc.php

```diff
@@ -599,7 +600,7 @@
 
 // Distributed Poller-Settings
 $config['distributed_poller']                            = false;
-$config['distributed_poller_name']                       = file_get_contents('/proc/sys/kernel/hostname');
+$config['distributed_poller_name']                       = trim(file_get_contents('/proc/sys/kernel/hostname'));
```

Comment:
Looking at my pollers it is purely registering them with new lines, this change means I'll see 4 pollers instead of two with the old ones never updating. If it's possible to push an SQL update out with this to strip \n off then it will be fine.